deepspeed/launcher/multinode_runner.py

```diff
@@ -34,7 +34,7 @@ def get_cmd(self, environment, active_resources):
         """Return the command to execute on node"""
 
     def add_export(self, key, var):
-        self.exports[key.strip()] = var.strip()
+        self.exports[key.strip()] = f"\"{var.strip()}\""
```

Comment:
@TideDra have you tested this?

I think an issue we have seen previously is if the key is supposed to contain quotes, we run into issues with this solution, right?

Reply:
I test this in my use case and it solves the bug I meet. I think this is a safe operation, since the `var` here is already treated as string, and using quotes just makes sure it is treated as string too when exported. If there was any issue caused by quotes previously, please let me know😀